modules/openapi-generator/src/main/java/org/openapitools/codegen/InlineModelResolver.java

```diff
@@ -956,6 +956,52 @@ private String addSchemas(String name, Schema schema) {
             name = inlineSchemaNameMapping.get(name);
         }
 
+        // If name only consists of underscores, and schema has chlidren models, create new name
+        if (name.replace("_", "") == "" & schema instanceof ComposedSchema) {
```

Comment:
use `.equals()` instead of `==` for string comparison

Reply:
fixed it!